src/tools/kata-ctl/Cargo.toml

```diff
@@ -19,6 +19,9 @@ serde_json = "1.0.85"
 thiserror = "1.0.35"
 privdrop = "0.5.2"
 nix = "0.25.0"
+rtnetlink = "0.11.0"
```

Comment:
This crate has been abandoned fwics:

https://github.com/little-dude/netlink

The readme for that repo suggests using https://github.com/orgs/rust-netlink, but it's unclear which of the [19 repos](https://github.com/orgs/rust-netlink/repositories) we're supposed to use instead :)

Possibly https://github.com/rust-netlink/rtnetlink, but that hasn't been uploaded to https://github.com/rust-netlink/rtnetlink fwics.

/cc @Tim-0731-Hzt as we're using that in the new [runtime-rs `resource` crate](https://github.com/kata-containers/kata-containers/blob/main/src/runtime-rs/crates/resource/Cargo.toml#L23).

Reply:
@DabinJang96 - fyi, this coment isn't a blocking one for your PR to land :smile: